modules/UI/recording/Recording.js

```diff
@@ -471,15 +470,21 @@ const Recording = {
             return;
         }
 
-        sendAnalyticsEvent(RECORDING_CLICKED);
+        sendAnalytics(createToolbarEvent('recording.button'));
```

Comment:
I wonder why this only gets logged when there is no dialog. Maybe it should always get logged and dialog present be an attribute?

Reply:
Good point, fixed